typing-indicator/src/main/java/com/udevel/widgetlab/sequences/RandomNoRepetitionSequence.java

```diff
@@ -0,0 +1,16 @@
+package com.udevel.widgetlab.sequences;
+
+/**
+ * Created by joragu on 6/21/2017.
+ */
+
+public class RandomNoRepetitionSequence extends RandomSequence {
+    @Override
+    public int nextIndex(int currentIndex, int numberOfElements) {
+        int nextValue;
+        do {
+            nextValue = super.nextIndex(currentIndex, numberOfElements);
```

Comment:
i thought about it as well, but, because it uses a normal distribution it "shouldn't" happen, but, i guess you are correct technically, i was thinking of adding a counter and then just finishing if it took more than like 5 tries or something.